lib/index.js

```diff
@@ -99,16 +98,17 @@ function StickyInlines(opts) {
    */
 
   function onSelect(event, change) {
-    const selection = findRange(window.getSelection(), change.value);
-    const selectionIsAtEndOfInline = change.value.focusInline &&
-      selection.focusOffset === change.value.focusInline.text.length;
+    if(!change.value.focusInline) return null
```

Comment:
With some modifications to anything that wasn't available, most of this is actually from: https://github.com/aha-app/slate-sticky-inlines/blob/slate-0.37/lib/index.js